src/forecast.coffee

```diff
@@ -271,7 +278,7 @@ module.exports = (robot) ->
   fetchForecast = (callback) ->
 
     forecastKey = process.env.HUBOT_FORECAST_KEY
-    exclude = 'hourly,daily,flags'
+    exclude = 'flags'
```

Comment:
If I recall, this was done on purpose to have the automatically fetched forecast not return the hourly or daily bits. I’d need to double-check that to be sure, however.